pylon_camera/include/pylon_camera/internal/impl/pylon_camera_gige.hpp

```diff
@@ -201,7 +201,7 @@ bool PylonGigECamera::applyCamSpecificStartupSettings(const PylonCameraParameter
                 // Check if gamma is available, print range
                 if ( !GenApi::IsAvailable(cam_->Gamma) )
                 {
-                    ROS_WARN("Cam gamma not available, will keep the default (auto).");
+                    ROS_INFO("Cam gamma not available, will keep the default (auto).");
```

Comment:
Why is this switched from WARN to INFO?

Reply:
There's a lot of warns that happen on startup which seem left over from debugging and in my opinion are not warn-worthy. We'll see what upstream thinks if/when I do a PR there.

Reply:
Gotcha.  Since we're upstreaming this, I'd say it's better to stay consistent with the existing codebase (if this were our code, I'd want all of the similar messages to be either all WARN's or all INFO's or DEBUG's and we could decide which to choose based on the context).  And honestly if you think these aren't errors maybe they should be DEBUG statements anyway?  If you feel strongly about this, I think the right place for it is probably a separate PR changing all of them (I think this was the only one I noticed changed).